pkg/vscrstatus/status.go

```diff
@@ -70,6 +70,9 @@ func UpdateStatus(ctx context.Context, clnt client.Client, vscr *v1beta1.Vertica
 		if vscr.Status.TarballName != vscrChgStatus.TarballName {
 			vscr.Status.TarballName = vscrChgStatus.TarballName
 		}
+		if vscr.Status.State != vscrChgStatus.State {
```

Comment:
Any advantage in check the prior state? Would it be simple to just remove the if checks totally.

Reply:
Yes just a bad habit. I will remove the checks.